packages/amplify-hosting-service/src/helpers/stack-helper.ts

```diff
@@ -105,7 +105,7 @@ export class StackHelper {
         spinner.start(`Deleting Amplify Console stack for ${envName} environment`);
         await cfnClient.deleteStack(params).promise().catch(err => {
             if (err.code === VALIDATION_ERROR) {
-                spinner.succeed(`Deleting Amplify Console stack for ${envName} environment completed.`);
+                spinner.succeed(`${envName} environment deletion complete.`);
```

Comment:
add "frontend" or "Amplify console" somewhere to differentiate this from the backend env.